package.json

```diff
@@ -17,6 +17,7 @@
     "@upstash/redis": "^1.30.0",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "dotenv": "^16.4.5",
```

Comment:
Nestjs already handles .env files, this dependency is not needed

Reply:
Yea fair, I was using it in a file tho even if it didnt do anything please @dpaulos6 when you remove a package remove everything!